HTML_Pages/list.html

```diff
@@ -0,0 +1,206 @@
+            </a> 
+            <i class="fa fa-remove text-primary sidenav-close-btn" id="close-side-nav"></i>
+        </div>
+        <div class="side-nav-div">
+            <p class="side-header">Getting started</p>
+            <a href="/HTML_Pages/docs.html#installation-page" class="btn-link text-primary side-list">Installation</a>
+            <a href="/HTML_Pages/docs.html#usage-page" class="btn-link text-primary side-list">Usage</a>
+        </div>
+        <div class="side-nav-div">
+            <p class="side-header">Components</p>
+            <a href="/HTML_Pages/alert.html" class="btn-link text-primary side-list focus-link">Alerts</a>
+            <a href="/HTML_Pages/avatar.html" class="btn-link text-primary side-list">Avatars</a>
+            <a href="/HTML_Pages/badge.html" class="btn-link text-primary side-list">Badges</a>
+            <a href="/HTML_Pages/button.html" class="btn-link text-primary side-list">Buttons</a>
+            <a href="/HTML_Pages/card.html" class="btn-link text-primary side-list">Cards</a>
+            <a href="/HTML_Pages/grid.html" class="btn-link text-primary side-list">Grids</a>
+            <a href="/HTML_Pages/image.html" class="btn-link text-primary side-list">Images</a>
+            <a href="/HTML_Pages/list.html" class="btn-link text-primary side-list">Lists</a>
+            <a href="/HTML_Pages/modal.html" class="btn-link text-primary side-list">Modals</a>
+            <a href="/HTML_Pages/navigation.html" class="btn-link text-primary side-list">Navigation</a>
+            <a href="/HTML_Pages/ratings.html" class="btn-link text-primary side-list">Ratings</a>
+            <a href="/HTML_Pages/sideNavigation.html" class="btn-link text-primary side-list">Side Navigation</a>
+            <a href="/HTML_Pages/slider.html" class="btn-link text-primary side-list">Slider</a>
+            <a href="/HTML_Pages/snackbar.html" class="btn-link text-primary side-list">Snackbar</a>
+            <a href="/HTML_Pages/textbox.html" class="btn-link text-primary side-list">Textboxes</a>
+            <a href="/HTML_Pages/textUtilities.html" class="btn-link text-primary side-list">Text Utilities</a>
+        </div>
+    </nav>
+    <div class="component-body" id="comp-body">
+        <h1 class="text-header">Lists</h1>
+        <p>Lists are continuous, vertical indexes of text or images.</p>     
+        <h3>Simple text list</h3>
+        <section class="comp-display-box-ver">
+            <div class="listpane">
+                <h4 class="list-header">Options</h4>
+                <ul class="lists">
+                    <li class="list-item">Single-line item</li>
+                    <li class="list-item">Single-line item</li>
+                    <li class="list-item">Single-line item</li>
+                </ul>
+            </div>
+        </section>
+        <section class="comp-code-box">
+            <pre>
+                <code class="html">
+                  
```

Comment:
You haven't added any code here

Reply:
Yes will add that after PR review, this basically contains the code that user will see